supabase/migrations/10_spec_ext.sql

```diff
@@ -95,8 +102,8 @@ select
 from live_specs l
 left outer join publication_specs p on l.id = p.live_spec_id and l.last_pub_id = p.pub_id
 left outer join connectors c on c.image_name = l.connector_image_name
-left outer join connector_tags t on c.id = t.connector_id and l.connector_image_tag = t.image_tag,
-lateral view_user_profile(p.user_id) u
+left outer join connector_tags t on c.id = t.connector_id and l.connector_image_tag = t.image_tag
+inner join user_profiles u on u.id = p.user_id;
```

Comment:
This should probably also be a left outer join.

it shouldn't matter in practice, but if a user is deleted from the system (we don't do this today, nor have a plan to) we'd still want this row to be surfaced in the view -- just with nulls for `last_pub_user_email` etc.